app/components/EditorHome/index.js

```diff
@@ -41,11 +42,18 @@ const EditorHome = ({ eventIDs }) => (
         <EventLink id={newEventID(eventIDs)} />
       </li>
     </ul>
+    <button onClick={() => publish({ events, community, conference })}>
+      Publish
```

Comment:
Currently each instance of the app can only publish to one bucket/location/whatever. We need to design and implement a draft system.